src/clj/ataru/applications/application_service.clj

```diff
@@ -150,6 +173,12 @@
                                              newest-form)
                                        (assoc :content [])
                                        (dissoc :organization-oid))
+          changes (if with-newest-form?
```

Comment:
Tässä funktiossa voisi olla paikallaan tehdä muutama refaktorointi tässä yhteydessä.

-  `form-in-application` ja `newest-form` voisivat sisältää suoraan `populate-form-fields` funktion läpi laitetut formit. molemmat formit syötetään ko. funktion läpi joka tapauksessa jossain vaiheessa.
- `changes` voidaan laskea ehdoitta, ja korvata `forms-differ?` tarkastamalla onko `changes` tyhjä vai ei.
- tämän jälkeen voidaan poistaa funktio `forms-differ?` joka on käytännössä `get-changed-field-ids` funktion duplikaatti.